src/Fitting/Stats.jl

```diff
@@ -0,0 +1,115 @@
+# =========================== src/Fitting/Stats.jl ===========================
+# Conventions:
+# - Rows = variables (p), columns = samples (n)
+# - This file exposes the StatsBase/Distributions interface for CopulaModel
+
+# ---------- 1) Utils ----------
+@inline _as_pxn(C::Copulas.Copula, U::AbstractMatrix) = (size(U,1) == length(C)) ? U : permutedims(U)
+@inline _as_pxn(p::Integer,         U::AbstractMatrix) = (size(U,1) == p)        ? U : permutedims(U)
+
+# ---------- 2) Log-likelihood (Distributions) ----------
+function Distributions.loglikelihood(C::Copulas.Copula, U::AbstractMatrix{<:Real})
```

Comment:
Cant this be defined easily as `sum(Distributions.logpdf(C, U)` ? Isnt it alreayd defined for us in Distributions.jl ??